alpha.py

```diff
@@ -34,6 +34,7 @@ class BaseAlphaSettings:
     All alpha settings should inherit this and re-define variables.
     """
     # required variables
+    alpha_name = None
```

Comment:
In prod, we will put all alphas of a user in the same folder.
That is, all files uploaded by the same user must have different name.
So the name of an AlphaSettings file is considered as the alpha's name.
Do we still need this variable?